### DISCUSSION TITLE
Portable bitmap support

### DISCUSSION THREAD

**Discussion by ynse01** [SixLabors/ImageSharp#1842] [Ideas] [answered]
I'm playing around with implementing support for the Portable Bitmap file format. These include the pgm, pbm, ppm and pam file extensions. See [wikipedia](https://en.wikipedia.org/wiki/Netpbm).

The format is relatively simple to implement. This fact causes it to be popular in scientific software. Could this be an useful addition to ImageSharp also?

If so, I would like to give it a try and make my code into a Pull Request.

**Comment by antonfirsov** [comment, marked as answer]
Considering the simplicity of the format, I don't see any disadvantages from adding it to the codebase, for me it looks like a nice addition. @JimBobSquarePants thoughts?

**Reply by JimBobSquarePants** [reply]
Yeah, fine by me.

**Reply by ynse01** [reply]
Working on this in PR #1851
